File: C/URI 1046.c
#include <stdio.h>
 
int main(){ 
    int Inicial, Final;

    scanf("%d %d", &Inicial, &Final);
    if (Inicial > Final) { 
        printf("O JOGO DUROU %d HORA(S)\n", 24 - (Inicial - Final));
    } else if (Final > Inicial) {
        printf("O JOGO DUROU %d HORA(S)\n", Final - Inicial);
    } else {
        printf("O JOGO DUROU 24 HORA(S)\n");
    }
    return 0;
}

File: Python/URI 1046.py
x = input().split()
inicial, final = x

inicial = int(x[0])
final = int(x[1])

if inicial < final:
    tempo = final - inicial
else:
    tempo = (24 - inicial) + final
print('O JOGO DUROU {} HORA(S)'.format(tempo))